### PR TITLE
Remove IRC, add community discord

### PR DESCRIPTION
**Pull request by urielsalis**

### CODE REVIEW

**Comment by LordRalex** [comment]
After consideration, we should leave the official discord as well. Please update the PR to keep both discords.

**Comment by urielsalis** [comment]
all done

File: help/technical-support-resources/index.html [hunk 1]
```diff
@@ -44,28 +44,26 @@ <h1>Technical support resources</h1>
...
+                <a href="https://discord.gg/58Sxm23" class="btn btn-success">Go →</a>
             </div>
             <div>
-                <h4><a href="https://minecraftirc.net/minecrafthelp/landing/">Community IRC support</a></h4>
-                <p>IRC, is a way for people to chat and hold conversations over the internet, no registrations required.
-                    For support with Minecraft;</p>
-                <p>You can join the <strong>#minecrafthelp</strong> channel, <strong>by going pressing the 'GO'
-                    button </strong>to your right, please be sure to let them know about the problem you have, and an
-                    Mojira issue ID, if you have one!</p>
+                <h4><a href="https://discord.gg/58Sxm23">Official Minecraft Discord</a></h4>
```
Comment (LordRalex) [inline help/technical-support-resources/index.html:60]: Your links are backwards

Comment (urielsalis) [reply help/technical-support-resources/index.html]: fixed

**Pull request merged**